diagnostic.md

```diff
@@ -36,4 +44,8 @@ Record your responses inside the fenced code blocks below each question.
 
     ```html
     <!-- your response here -->
+    {{#each my-phone.numbers as |number|}}
+        {{my-contact/my-phone number=number}}
+    {{/each}}
```

Comment:
We access the object property as `contact.phones`.  you don't need to have `my-contact/` in the sub-component.  So we would write it like this:
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```